packages/manifest-to-bicep-extension/src/manifest.ts

```diff
@@ -19,6 +19,7 @@ export interface Schema {
   type: 'string' | 'object' | 'integer' | 'boolean'
```

Comment:
I think this would need to be updated as well?

```suggestion
  type: 'string' | 'object' | 'integer' | 'boolean' | 'object'
```